File: 6_linked_list_and_arrays/5_remove_duplicates_from_sorted_array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        n = len(nums)
        curr = 1
        for i in range(1, n):
            if nums[i] == nums[i-1]:
                continue
            nums[curr] = nums[i]
            curr += 1
        return curr

File: 07_linked_list_and_arrays/5_remove_duplicates_from_sorted_array.rs
impl Solution {
    pub fn remove_duplicates(nums: &mut Vec<i32>) -> i32 {
        let n = nums.len();
        let mut curr = 1;

        for i in 1..n {
            if nums[i] == nums[i - 1] {
                continue;
            }
            nums[curr] = nums[i];
            curr += 1;
        }
        curr as i32
    }
}
